Add Doc Page to list where USD is supported (#2095)
PixarAnimationStudios/USD: docs/support.rst

to Maya as well.

Blender
=======

`Blender <https://www.blender.org>`_ is a free and open source 3D creation suite.

`Blender USD Documentation <https://docs.blender.org/manual/en/latest/files/import_export/usd.html>`_

Foundry
=======

The Foundry are a developer of 3D Content Creation, Lighting and Compositing software.

Katana
------

`Katana <https://www.foundry.com/products/katana>`_ is  look development and lighting package.

Katana has `open source USD plugins <https://learn.foundry.com/katana/dev-guide/Plugins/KatanaUSDPlugins/index.html>`_ as
well as a support for `Hydra render delegates <https://learn.foundry.com/katana/dev-guide/Plugins/HydraRenderDelegates/index.html>`_.

Modo
----

`Modo <https://www.foundry.com/products/modo>`_ is a general purpose 3D package.

Nuke
----

`Nuke <https://www.foundry.com/products/nuke-family/nuke>`_ is a compositing package with support for both USD and Hydra.

`Nuke documentation <https://learn.foundry.com/nuke/content/comp_environment/3d_compositing/usd.html>`_

ZBrush
------

`ZBrush <https://www.maxon.net/en/zbrush>`_ is a 3D sculpting and painting application.

> `ZBrush USD documenation <http://docs.pixologic.com/user-guide/zbrush-other-programs/usd-format/>`_

Reviewer: This could be under Maxon category, since Zbrush is owned by them now.

Author: Ah yeah you're right. I meant for it to be under Maxon but accidentally put it under Foundry

> Houdini

Reviewer: Shouldn't this be the company name for the sake of consistency ? (same for Nvidia as opposed to Omniverse, and Epic games for Unreal engine etc...)
```suggestion
SideFX
```
Or you simply use company name only when it's related to multiple packages ?

Author: I was only using Company names when they had multiple sections under them, but I'm open to change that if people feel having the company name as the top level should always make sense.

In which case the question would be:
1. Do single product companies have a nested section under them for the product? Or do we just call it `SideFX Houdini` and `NVIDIA Omniverse`?
2. If you do nested sections, do you do the same for Blender and Unity?

(I might totally be overthinking this...)

Author: actually, giving it more thought, the least churn (in case companies add more stuff) is to list it as `SideFX Houdini` and `NVIDIA OmniVerse`.

Reviewer: That is what I thought too, and you're right. 
I think calling them `SideFX Houdini` and `NVIDIA Omniverse` instead can solve both problems, I'm thinking for example for someone who doesn't know Omniverse, it's useful to have `NVIDIA` in the section name.

EDIT : and for Blender or Unity, they're already known mostly in those names. So I think that can work.

Author: Yep. Just updated the PR

=======

`Houdini <https://www.sidefx.com/products/houdini/>`_ is a 3D package with a focus on procedural content and effects.

Houdini includes `Solar <https://www.sidefx.com/products/houdini/solaris/>`_ to create composed USD content.

`SideFX USD Documentation <https://www.sidefx.com/docs/houdini/solaris/usd.html>`_

Maxon
=====

Maxon are a developer of 3D content creation, rendering and motion graphics software.

Cinema 4D
---------

`Cinema 4D <https://www.maxon.net/en/cinema-4d>`_ is a general purpose 3D package.

Redshift
--------

`Redshift <https://www.maxon.net/en/redshift>`_ is a GPU-accelerated path-traced renderer.

Redshift supports USD and is also available as a Hydra delegate for rendering integration.

Omniverse
=========

`NVIDIA Omniverse <https://www.nvidia.com/en-us/omniverse/>`_ is a platform for creating and operating metaverse applications.
It is based on USD.

`NVIDIA USD Documentation <https://www.nvidia.com/en-us/omniverse/usd/>`_

Unity
=====

`Unity <https://unity.com>`_ is a real time 3D engine and editor.
It includes an `open source USD Package <https://github.com/Unity-Technologies/usd-unity-sdk>`_.

> `Unity USD Documentation <https://docs.unity3d.com/2020.1/Documentation/Manual/com.unity.formats.usd.html>`_

Reviewer 2: I think this is a better/more direct link to the docs for the most recent package release:
https://docs.unity3d.com/Packages/com.unity.formats.usd@3.0/manual/index.html

Isotropix Clarisse
==================

`Clarisse <https://www.isotropix.com/products/clarisse/ifx>`_ is a 3D application for look development, lighting and rendering.

Maxon
=====

Maxon are a developer of 3D content creation, rendering and motion graphics software.

Cinema 4D
---------

`Cinema 4D <https://www.maxon.net/en/cinema-4d>`_ is a general purpose 3D package.

Redshift
--------

`Redshift <https://www.maxon.net/en/redshift>`_ is a GPU-accelerated path-traced renderer.

Redshift supports USD and is also available as a Hydra delegate for rendering integration.

ZBrush
------

`ZBrush <https://www.maxon.net/en/zbrush>`_ is a 3D sculpting and painting application.

`ZBrush USD documenation <http://docs.pixologic.com/user-guide/zbrush-other-programs/usd-format/>`_

NVIDIA Omniverse
=================

`NVIDIA Omniverse <https://www.nvidia.com/en-us/omniverse/>`_ is a platform for creating and operating metaverse applications.
It is based on USD.

`NVIDIA USD Documentation <https://www.nvidia.com/en-us/omniverse/usd/>`_

Omniverse also adds their own USD support to the following applications:

* 3ds Max
* Archicad
* Character Creator
* Creo

> * iClone
> * ParaViw

Reviewer: They also got a maya connector !
```suggestion
* iClone
* Maya
* ParaViw
```

EDIT : I noticed there is Unreal engine connector that's missing too ^^'

Author: Ah good catch. I found their list of connectors and linked that instead. I then pruned the list to just the items that aren't listed elsewhere so as to keep the list small but still show that there is a way to get these applications into a persons pipeline.

===============
Support for USD
===============

USD has support in many 3D Content Creation Applications and Ecosystems.

This page lists several of them, but is not fully comprehensive.

.. include:: rolesAndUtils.rst
.. include:: <isonum.txt>

Copyright |copy| 2018, Pixar Animation Studios,  *version 1.2*

.. contents:: :local:

3Delight
========

`3Delight <https://www.3delight.com>`_ is a path-traced renderer that supports USD and has
a `Hydra Delegate <https://gitlab.com/3Delight/HydraNSI>`_.

Adobe
=====

Adobe is a developer of several content generation applications. Their Substance suite of applications are geared
towards 3D content creation

Substance 3D Painter
--------------------

> 

Reviewer: Oh, almost forgot about this one !

```suggestion

Left Angle Autograph
=====

`Autograph <https://www.left-angle.com/#page=95>`_ is a compositing and motion design package that comes with a 3d system based on USD.

```

Plant Factory
-------------

`Plant Factory <https://info.e-onsoftware.com/plantfactory/overview>`_ is a vegetation generation software.

Vue
---

`Vue <https://info.e-onsoftware.com/vue/overview>`_ is world building application.

Gaffer
======

`Gaffer <https://www.gafferhq.org>`_ is an `open source <https://github.com/GafferHQ/gaffer>`_ , node based application for lighting and compositing.

Golaem
======

`Golaem <https://golaem.com>`_ is a crowd simulation program with an `open source, USD plugin <https://github.com/Golaem/GolaemForUSD>`_
that allows viewing Golaem crowd caches within USD.

Guerilla Render
===============

`Guerilla Render <http://guerillarender.com>`_ is a 3D renderer.

Foundry
=======

The Foundry are a developer of 3D Content Creation, Lighting and Compositing software.

Katana
------

`Katana <https://www.foundry.com/products/katana>`_ is  look development and lighting package.

Katana has `open source USD plugins <https://learn.foundry.com/katana/dev-guide/Plugins/KatanaUSDPlugins/index.html>`_ as
well as a support for `Hydra render delegates <https://learn.foundry.com/katana/dev-guide/Plugins/HydraRenderDelegates/index.html>`_.

Mari
----

> `Mari <https://github.com/Golaem/GolaemForUSD>`_ is a 3D texture painting application.

Reviewer 3: ```suggestion
`Mari <https://www.foundry.com/products/mari>`_ is a 3D texture painting application.
```
Incorrect URL

> 

Reviewer 3: ```suggestion
Mari has `open source USD plugins <https://github.com/TheFoundryVisionmongers/MariUsdPlugins>`_.

```
Here is Mari's github link to the USD plugins that are shipped with each release

Modo
----

`Modo <https://www.foundry.com/products/modo>`_ is a general purpose 3D package.

Nuke
----

`Nuke <https://www.foundry.com/products/nuke-family/nuke>`_ is a compositing package with support for both USD and Hydra.

`Nuke documentation <https://learn.foundry.com/nuke/content/comp_environment/3d_compositing/usd.html>`_

Isotropix Clarisse
==================

`Clarisse <https://www.isotropix.com/products/clarisse/ifx>`_ is a 3D application for look development, lighting and rendering.

> 

Reviewer: Maybe it's worth mentioning Dreamworks' Moonray too ?

```suggestion

Dreamworks Moonray
==================

`Moonray <https://openmoonray.org/>`_ is an open-source renderer that comes with a USD Hydra render delegate.

```

J Cube
======

J Cube are a developer of 3D software and cloud services.

Multiverse for Maya
-------------------

`Multiverse <https://j-cube.jp/solutions/multiverse/>`_ is a USD plugin for Maya with a range of pipeline features.

Muse
----

`Muse <https://j-cube.jp/solutions/muse/>`_ is a standalone USD Editor

> 

Reviewer: Oh, almost forgot about this one !

```suggestion

Left Angle Autograph
=====

`Autograph <https://www.left-angle.com/#page=95>`_ is a compositing and motion design package that comes with a 3d system based on USD.

```

Shapr3D
=======

`Shapr3D <https://www.shapr3d.com>`_ is a CAD application that supports USD export on iPads, Mac and Windows.

SideFX Houdini
==============

`Houdini <https://www.sidefx.com/products/houdini/>`_ is a 3D package with a focus on procedural content and effects.

Houdini includes `Solar <https://www.sidefx.com/products/houdini/solaris/>`_ to create composed USD content.

`SideFX USD Documentation <https://www.sidefx.com/docs/houdini/solaris/usd.html>`_

SynthEyes
=========

`SynthEyes <https://www.ssontech.com>`_ is a match move application.

Tilt Brush
==========

`Tilt Brush <https://www.tiltbrush.com>`_ is a VR painting application.

Unity
=====

`Unity <https://unity.com>`_ is a real time 3D engine and editor.
It includes an `open source USD Package <https://github.com/Unity-Technologies/usd-unity-sdk>`_.

`Unity USD Documentation <https://docs.unity3d.com/Packages/com.unity.formats.usd@3.0/manual/index.html>`_

Unreal Engine
=============

`Unreal Engine <https://www.unrealengine.com/en-US>`_ is a real time 3D engine and editor.

`Unreal USD Documentation <https://docs.unrealengine.com/5.0/en-US/universal-scene-description-usd-in-unreal-engine/>`_

> 

Reviewer: I think usdtweak is worth adding too, since it's becoming a very feature rich USD focused editor.

```suggestion

Usdtweak
=============

`usdtweak <https://github.com/cpichard/usdtweak>`_ is a free and open source editor for USD.

```

Vicon Shogun
============

`Vicon Shogun <https://www.vicon.com/software/shogun/>`_ is a motion capture application.

Wizart
======

> `Wizart <https://wizanim.com/technology/>`_ is a USD based general purpose 3D application.

Reviewer: I think this is a better link for the Wizart DCC Platform,  and it's already in english

```suggestion
`Wizart DCC Platform <https://wizartsoft.com/>`_ is a USD based general purpose 3D application.
```
